packages/flame_forge2d/example/lib/joint_sample.dart

```diff
@@ -0,0 +1,68 @@
+import 'dart:math';
+
+import 'package:flame/extensions.dart';
+import 'package:flame/game.dart';
+import 'package:flame/input.dart';
+import 'package:flame_forge2d/body_component.dart';
+import 'package:flame_forge2d/flame_forge2d.dart';
+import 'package:flame_forge2d/forge2d_game.dart';
+import 'package:forge2d/forge2d.dart';
+
+import 'balls.dart';
+import 'boundaries.dart';
+
+class CircleShuffler extends BodyComponent {
+  final Ball ball;
+
+  CircleShuffler(this.ball);
+
+  @override
+  Body createBody() {
+    final bodyDef = BodyDef()
+      ..type = BodyType.dynamic
+      ..position = ball.body.position.clone();
+    const numPieces = 5;
+    const radius = 6.0;
+    final body = world.createBody(bodyDef);
+
+    for (var i = 0; i < numPieces; i++) {
+      final xPos = radius * cos(2 * pi * (i / numPieces));
+      final yPos = radius * sin(2 * pi * (i / numPieces));
```

Comment:
I've seen this pattern here and there; makes me wonder if a `Vector2.cis` constructor (`cos + i*sin`) would be too overkill...